src/global/Types.ts

```diff
@@ -149,11 +161,35 @@ export interface AnnotationTemplate {
     },
     target: {
         id: string,
-        indvProp: string,
-        type: string
-    },
+        indvProp?: string,
+        type: string,
+    }
 };
 
+export interface ImageAnnotationTemplate {
+    type: string,
+    motivation: string,
+    body: {
+        type: string,
+        values: string[]
+    },
+    target: {
```

Comment:
Looks duplicate

Reply:
Yeah kinda, I will replace this in the upcoming PR for the implementation of the Annotation model